packages/host/app/services/card-service.ts

```diff
@@ -73,7 +73,12 @@ export default class CardService extends Service {
     let card = await this.api.createFromSerialized(resource, doc, relativeTo, {
       loader: this.loaderService.loader,
     });
-    await this.api.recompute(card);
+    // it's important that we absorb the field async here so that glimmer won't
+    // encounter NotReady errors, since we don't have the luxury of the indexer
+    // being able to inform us of which fields are used or not at this point.
+    // (this is something that the card compiler could optimize for us in the
+    // future)
+    await this.api.recompute(card, { recomputeAllFields: true });
```

Comment:
the server has already included the computed field values (albiet in wire format)--it would be good if we can reuse this work that the server did. perhaps an attribute that can inform us if the computeds are valid/consistent with the rest of the fields?